build/Targets/Settings.props

```diff
@@ -101,6 +102,12 @@
   <!-- Unix specific settings -->
   <PropertyGroup Condition="'$(OS)' != 'Windows_NT'">
     <RoslynDebugType>portable</RoslynDebugType>
+
+    <!-- Presently can't build anything with a net46 TargetFramework on non-Windows. Disabling
+         for now. 
+        
+         https://github.com/dotnet/roslyn/issues/20932 -->
```

Comment:
I linked the SDK issue tracking this (https://github.com/dotnet/sdk/issues/826) in your bug.